src/oc/auth/resources/user.clj

```diff
@@ -137,6 +137,15 @@
     false))
 
 ;; ----- User CRUD -----
+(def default-user-image "/img/ML/happy_face_red.svg")
```

Comment:
@thepug you don't use the `default-user-image` here on server side so you can move this inside the `other-user-images` (maybe call it something better) and directly shuffle this array instead of using conj.
Makes sense?